intro-sql-access.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: 'Query your database using SQL'
+description: 'Run SQL queries over HTTP directly in PostgreSQL'
+author: Noémi Ványi
+date: 08-31-2023
+published: false
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/intro-sql-access-over-http-cover.png
+  alt: SQL access to Xata
+slug: sql-over-http
+---
+
+We are excited to announce that Xata now offers direct SQL access to your database, in addition to the Xata API. This allows you to express arbitrary complex SQL queries and use SQL builders like Kysley or Drizzle. You can play with the new SQL support in the built-in Playground, which we have extended to support SQL.
```

Comment:
Suggestion:

"As part of our [launch week](https://xata.io/blog/launch-week-august-2023), we are excited to announce..."

Suggestion:

"You can also now save and run SQL queries in our Playground, which we have extended to support SQL."

I mentioned this in an earlier comment, but If we're opening with it here, we should also probably highlight either through a screenshot or dedicated section.